### ISSUE TITLE
"Lighthouse is warming up..." 

### ISSUE THREAD

**Issue by Th3S4mur41** [MicrosoftEdge/DevTools#169]
and never goes beyond the following screen in Edge Canary
![image](https://github.com/MicrosoftEdge/DevTools/assets/5543595/61b3dd30-ad04-4a51-9a7b-955eb7ed6205)

It is working fine in the stable Edge on the same machine though.

Probably related to [the same issue in Chrome](https://github.com/GoogleChrome/lighthouse/issues/15026)

AB#45665567

**Comment by captainbrosset** [comment]
Thank you for reporting this. I was able to reproduce this locally on Edge Canary 117 on Windows 11.
I will make sure this shows up on our team's backlog for investigation/prioritization/fixing.

**Comment by yksht** [comment]
I have same issue.

Microsoft Edge
Version 115.0.1901.183 (Official build) (64-bit)

Windows 11 Pro 22H2
OS Build 22621.1992

**Comment by XFox111** [comment]
Same here. Lighthouse hangs on "warming up" on two of my PCs and inside Windows Sandbox. Tried to restart, force kill, repair installation, use incognito/guest profiles - same result.

Latest Edge release (Stable branch):
115.0.1901.188 (Official build) (64-bit)

Windows 11 Pro 22H2
OS build 22621.2070
Windows Feature Experience Pack 1000.22659.1000.0

**Comment by spanishgringo** [comment]
Same issue Version 115.0.1901.188 (Official build) (arm64)
MacOs Ventura 13.5 with M2

**Comment by Randomblock1** [comment]
Same issue
Edge 115.0.1901.188 (Official build) (64-bit)
Windows 11 Pro 22H2
OS build	22621.2070

**Comment by J6ey** [comment]
Frustrated.
Version 115.0.1901.188 (Official build) (64-bit)

**Comment by mustafa-barakat** [comment]
Version 115.0.1901.188 (Official build) (64-bit)
Version 117.0.2024.0 (Official build) canary (64-bit)

Both aren't working with any website; chrome is performing these tests on the same websites fine.

**Comment by PIndividual** [comment]
Same here
Version 115.0.1901.200 (Official build) (64-bit)
Windows 10 Pro for Workstations 22H2

**Comment by Ashbri4** [comment]
What about a link for android 14

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by betorina** [comment]
This problem is unfortunately still happening... 
it's not working in either 114 and 115 (tried all possible tips/fixes, and a also an Edge Beta version)

Version 115.0.1901.200 (Official build) (arm64)
Mac M2 Pro - Ventura 13.5

**Comment by IndexingGitHub** [comment]
Same here on Edge Dev

**Comment by glimpsedchaos** [comment]
Same issue - Edge Version 115.0.1901.183

**Comment by Th3S4mur41** [comment]
It's working again for me in Edge 117.0.2039.0 (Official build) canary (64-bit) on Windows 11
Still broken on the same system on stable Edge (115.0.1901.203)

**Comment by ankityadav4** [comment]
This problem is still happening.

**Comment by XFox111** [comment]
> It's working again for me in Edge 117.0.2039.0 (Official build) canary (64-bit) on Windows 11 Still broken on the same system on stable Edge (115.0.1901.203)

I can confirm that. It works on Canary branch again, still broken on Stable

**Comment by mabdelfattah** [comment]
Not working - Version 116.0.1938.54 (Official build) Beta (arm64) - MacOS

**Comment by captainbrosset** [comment]
Thanks for your patience. Our engineering team is still investigating this. A preliminary fix went into 117 already.

**Comment by betorina** [comment]
> A preliminary fix went into 117 already.

Please test... before giving 'good' news. 
Still no luck – lighthouse still hangs – on Version 117.0.2045.1  (Mac M2)

**Comment by XFox111** [comment]
On 118 Canary still doesn't work. But at least it now shows some details:

```
Channel: DevTools
Initial URL: https://www.google.com/
Chrome Version: 118.0.0.0
Stack Trace: Error: Could not create page handle for the target page
    at r (devtools://devtools/bundled/entrypoints/lighthouse_worker/lighthouse_worker.js?remoteBase=https%3A%2F%2Fdevtools.azureedge.net%2Fserve_file%2F%400d366432b851a89eb199fb26c56f471f25bab39c%2F:1:2375)
    at async self.onmessage (devtools://devtools/bundled/entrypoints/lighthouse_worker/lighthouse_worker.js?remoteBase=https%3A%2F%2Fdevtools.azureedge.net%2Fserve_file%2F%400d366432b851a89eb199fb26c56f471f25bab39c%2F:1:2973)
```

<details>
<summary><code>edge://version</code> on 118 Canary</summary>

Microsoft Edge | 118.0.2048.0 (Official build) canary (64-bit)
-- | --
Revision | 0d366432b851a89eb199fb26c56f471f25bab39c
Chromium version | 118.0.5940.0
Operating system | Windows 11 Version 22H2 (Build 22621.2134)
JavaScript | V8 11.8.1
User agent | Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/118.0.0.0 Safari/537.36 Edg/118.0.0.0
Command-line | "C:\Users\WDAGUtilityAccount\AppData\Local\Microsoft\Edge SxS\Application\msedge.exe" --flag-switches-begin --flag-switches-end --disable-nacl
Executable path | C:\Users\WDAGUtilityAccount\AppData\Local\Microsoft\Edge SxS\Application\msedge.exe
Profile path | C:\Users\WDAGUtilityAccount\AppData\Local\Microsoft\Edge SxS\User Data\Default
Active Variations | 24f921dd-d8b87158 8fab779b-d8b87158 b1800f26-d8b87158 ca24a5e1-d8b87158 286fdba8-d8b87158 2a54859-d8b87158 ea5e110f-d8b87158 133488ad-d8b87158 14007ca6-d8b87158 4c14a889-d8b87158 c2c6128f-d8b87158 f0e7a25e-d8b87158 c4f09439-d8b87158 cd0a2e45-ca7d8d80 ddef57c-d8b87158 2dcf9fe5-ca7d8d80 b1c22430-d8b87158 8d005f24-d8b87158 8dfb8021-d8b87158 d794b028-ca7d8d80 840681a4-d8b87158 76bf68d6-d8b87158 5683d3ed-ca7d8d80 572a7c6f-ca7d8d80 9ab3413-d8b87158 e1512345-d8b87158 66008988-ca7d8d80 cd3eaa09-ca7d8d80 1e92a065-d8b87158 a117e350-d8b87158 5c33b2b4-d8b87158 5d72e02a-ca7d8d80 c0856192-d8b87158 bb06bacd-d8b87158 90923109-d8b87158 a7896e11-ca7d8d80 a0596364-d8b87158 eb1bc628-d8b87158 ce2685d8-d8b87158 5b14374-d8b87158 7ae07155-ca7d8d80 4ae8b1dc-d8b87158 18e0a776-ca7d8d80 8026ad47-d8b87158 12c77193-d8b87158 ca0f6b5a-d8b87158 6c73f6ba-d8b87158 efe9c433-d8b87158 9cf91d25-ca7d8d80 17386f0-d8d26f5e 3524e495-d8d26f5e b75782da-d8d26f5e c9615a18-d8b87158 360e8bdf-ca7d8d80 9ff0f0c2-d8b87158 4ea303a6-ca7d8d80 30028a54-d8b87158 5fe87c3b-d8b87158 afd40c01-d8b87158 5329947e-d8b87158 d01bec90-ca7d8d80 2baaf711-d8b87158 f5d63c06-d8b87158 e9a1741-d8b87158 251623eb-d8b87158 66594ebe-d8b87158 6e1ac732-d8b87158 b4cbfdb5-d8b87158 1225f331-d8b87158 2422dd57-d8b87158 97dfc631-d8b87158 13d784cb-d8b87158 e9e1c3ba-d8b87158 bdf5b863-d8b87158 18e1bdad-ca7d8d80 ce67070e-ca7d8d80 1969e5c-d8b87158 ffca31e9-d8b87158 793340c0-d8d26f5e 50ed44a2-d8d26f5e 31e6ccb1-d8d26f5e 97d558ba-d8d26f5e e3f911b8-d8d26f5e 5a7ead15-d8d26f5e c1aadbf9-d8d26f5e d3706a2c-d8d26f5e f8f08c07-d8d26f5e f417542f-d8d26f5e 700c5884-d8d26f5e 70ceb6a7-d8d26f5e b25b7c7e-d8d26f5e aebb63e3-d8d26f5e b1fa74fa-d8d26f5e ccdfedf0-d8d26f5e 62f0d045-d8d26f5e aaad07de-d8d26f5e cdf29b77-d8d26f5e 59aa47b7-d8d26f5e 80bdf841-d8d26f5e f174dd26-d8d26f5e af89d238-d8d26f5e 63a1a746-d8d26f5e e241c53-d8d26f5e be6ca09b-d8d26f5e d2280f33-d8d26f5e e1af6e21-d8d26f5e 224053f9-d8d26f5e fbcd9197-d8d26f5e 147fd434-d8d26f5e 34485df7-d8d26f5e 38f16af1-d8d26f5e 4fe8b5de-d8d26f5e 2dab07dc-d8d26f5e 7accdcfa-d8d26f5e a408c5a6-d8d26f5e 4a8c62b4-d8d26f5e f5bba047-d8d26f5e 8891cb38-d8d26f5e a4ad23c5-d8d26f5e 611f1798-d8d26f5e 916bd790-d8d26f5e 8db5d24b-d8d26f5e 24e02e3-d8d26f5e ba2dded3-d8d26f5e 4f10b653-d8d26f5e a9140717-d8d26f5e e41c7906-d8d26f5e 8287f006-d8d26f5e 86d34dfc-d8d26f5e b1508168-d8d26f5e 23b45991-d8d26f5e 5e7aef88-d8d26f5e 4c7ac06c-d8d26f5e b322f040-d8d26f5e c6fdbeb2-d8d26f5e b7122a2e-d8d26f5e 29bf163-d8d26f5e 91b1889a-d8d26f5e 8d4b491c-d8d26f5e eaab553f-d8d26f5e cc4725b-d8d26f5e 6d5d2fe5-d8d26f5e b54d4573-d8d26f5e e0e61be6-d8d26f5e ed7b7a-d8d26f5e 1fc6c46e-d8d26f5e 69168e5a-d8d26f5e 4d4c330-d8d26f5e 4d3051ff-d8d26f5e f08cdbda-d8d26f5e db94232b-d8d26f5e 31dae2b7-d8d26f5e 8f66cc60-d8d26f5e 76eb9762-d8d26f5e 3d0e5fda-d8d26f5e 6e79aed6-d8d26f5e a185773f-d8d26f5e 80011f72-d8d26f5e e15deb37-d8d26f5e d37cb105-d8d26f5e c5cf1f99-d8d26f5e 72b74988-d8d26f5e daf8ad14-d8d26f5e abf13ad5-d8d26f5e 3113f9a-d8d26f5e dac2b8ab-d8d26f5e 5b00f587-d8d26f5e 19ea4f9d-d8d26f5e 512f1b6-d8d26f5e 851d4b5a-d8d26f5e c6a889f3-d8d26f5e aa2cc182-d8d26f5e e75f3d0d-d8d26f5e e35bd663-d8d26f5e b98ebbe6-d8d26f5e 4777da1c-d8d26f5e f510ede2-d8d26f5e 3f44e40-d8d26f5e d205504-d8d26f5e afc004b9-d8d26f5e 63e4737a-d8d26f5e 7edb7174-d8d26f5e c74e4c0d-d8d26f5e d557113c-d8d26f5e e4305f1-d8d26f5e 4b8fc442-d8d26f5e 237827c9-d8d26f5e b44c7049-d8d26f5e 5c9bda13-d8d26f5e 7c08e226-d8d26f5e 66d41c9d-d8d26f5e dcec42b9-d8d26f5e 45937cf-d8d26f5e dfb6ddc4-d8d26f5e 172199e7-d8d26f5e 24062188-d8d26f5e 687bee50-d8d26f5e f6ee8563-d8d26f5e 9e562fe6-d8d26f5e 7089f363-d8d26f5e baad6c16-d8d26f5e d426c084-d8d26f5e bd45ab20-d8d26f5e ec314542-d8d26f5e 760043ac-d8d26f5e 1fcb7bba-d8d26f5e 54e86e30-d8d26f5e 3fabe990-d8d26f5e b8163a1b-d8d26f5e 58446682-d8d26f5e b8be49ce-d8d26f5e 39fc393d-d8d26f5e 646696ca-d8d26f5e ba12d201-d8d26f5e 233fe1a-d8d26f5e 6d2b3687-d8d26f5e cf5b26ea-d8d26f5e 37def2a8-d8d26f5e ef45cf35-d8d26f5e 9c8712ec-d8d26f5e a9e98ced-d8d26f5e baa1ee78-d8d26f5e 3c0c1a57-d8d26f5e acbb1416-d8d26f5e 121ec64c-d8d26f5e 654bbccb-d8d26f5e 677e7ede-d8d26f5e 792a56ff-d8d26f5e cb80617e-d8d26f5e ca8b9f78-d8d26f5e a37d716f-d8d26f5e 381171fb-d8d26f5e b55f4b28-d8d26f5e fa505c3c-d8d26f5e 868a9bd9-d8d26f5e bcac1c59-d8d26f5e 7e17ac98-d8b87158 4a9a61df-d8b87158

</details>

Still freezes with no additional info on 117 Dev 

<details>
<summary><code>edge://version</code> on 117 Dev</summary>

Microsoft Edge | 117.0.2045.1 (Official build) dev (64-bit)
-- | --
Revision | a303c3cbe0ebf62c8bdcbaa6e64f8e995c08b5b2
Chromium version | 117.0.5938.0
Operating system | Windows 11 Version 22H2 (Build 22621.2134)
JavaScript | V8 11.7.24
User agent | Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/117.0.0.0 Safari/537.36 Edg/117.0.0.0
Command-line | "C:\Program Files (x86)\Microsoft\Edge Dev\Application\msedge.exe" --flag-switches-begin --flag-switches-end --disable-nacl
Executable path | C:\Program Files (x86)\Microsoft\Edge Dev\Application\msedge.exe
Profile path | C:\Users\WDAGUtilityAccount\AppData\Local\Microsoft\Edge Dev\User Data\Default
Active Variations | 24f921dd-d8b87158 8fab779b-d8b87158 b1800f26-d8b87158 286fdba8-d8b87158 2a54859-d8b87158 133488ad-d8b87158 14007ca6-d8b87158 4c14a889-d8b87158 c2c6128f-d8b87158 f0e7a25e-d8b87158 c4f09439-d8b87158 cd0a2e45-ca7d8d80 ddef57c-d8b87158 2dcf9fe5-ca7d8d80 b1c22430-d8b87158 8d005f24-d8b87158 d794b028-ca7d8d80 840681a4-d8b87158 66008988-ca7d8d80 cd3eaa09-ca7d8d80 1e92a065-d8b87158 5c33b2b4-d8b87158 bb06bacd-d8b87158 90923109-d8b87158 a7896e11-ca7d8d80 a0596364-d8b87158 eb1bc628-d8b87158 ce2685d8-d8b87158 5b14374-d8b87158 7ae07155-ca7d8d80 18e0a776-ca7d8d80 8026ad47-d8b87158 ca0f6b5a-d8b87158 6c73f6ba-d8b87158 efe9c433-d8b87158 9cf91d25-ca7d8d80 17386f0-d8d26f5e b75782da-d8d26f5e 6aa1871c-ca7d8d80 c9615a18-d8b87158 360e8bdf-ca7d8d80 9ff0f0c2-d8b87158 4ea303a6-ca7d8d80 30028a54-d8b87158 5fe87c3b-d8b87158 afd40c01-d8b87158 5329947e-d8b87158 d01bec90-ca7d8d80 2baaf711-d8b87158 f5d63c06-d8b87158 e9a1741-d8b87158 251623eb-d8b87158 66594ebe-d8b87158 6e1ac732-d8b87158 b4cbfdb5-d8b87158 1225f331-d8b87158 2422dd57-d8b87158 13d784cb-d8b87158 bdf5b863-d8b87158 18e1bdad-ca7d8d80 ce67070e-ca7d8d80 1969e5c-d8b87158 ffca31e9-d8b87158 793340c0-d8d26f5e 31e6ccb1-d8d26f5e f8f08c07-d8d26f5e f417542f-d8d26f5e 70ceb6a7-d8d26f5e b25b7c7e-d8d26f5e b1fa74fa-d8d26f5e ccdfedf0-d8d26f5e aaad07de-d8d26f5e cdf29b77-d8d26f5e 59aa47b7-d8d26f5e 80bdf841-d8d26f5e f174dd26-d8d26f5e af89d238-d8d26f5e 63a1a746-d8d26f5e e241c53-d8d26f5e d2280f33-d8d26f5e fbcd9197-d8d26f5e 2dab07dc-d8d26f5e 7accdcfa-d8d26f5e a408c5a6-d8d26f5e f5bba047-d8d26f5e 8891cb38-d8d26f5e a4ad23c5-d8d26f5e 611f1798-d8d26f5e 916bd790-d8d26f5e 8db5d24b-d8d26f5e 24e02e3-d8d26f5e 4f10b653-d8d26f5e a9140717-d8d26f5e 86d34dfc-d8d26f5e b1508168-d8d26f5e 4c7ac06c-d8d26f5e b322f040-d8d26f5e c6fdbeb2-d8d26f5e 29bf163-d8d26f5e eaab553f-d8d26f5e cc4725b-d8d26f5e ed7b7a-d8d26f5e 1fc6c46e-d8d26f5e 69168e5a-d8d26f5e 4d4c330-d8d26f5e 4d3051ff-d8d26f5e f08cdbda-d8d26f5e db94232b-d8d26f5e e15deb37-d8d26f5e c5cf1f99-d8d26f5e 72b74988-d8d26f5e daf8ad14-d8d26f5e dac2b8ab-d8d26f5e 19ea4f9d-d8d26f5e 512f1b6-d8d26f5e 851d4b5a-d8d26f5e c6a889f3-d8d26f5e e75f3d0d-d8d26f5e b98ebbe6-d8d26f5e 3f44e40-d8d26f5e d205504-d8d26f5e afc004b9-d8d26f5e 63e4737a-d8d26f5e d557113c-d8d26f5e e4305f1-d8d26f5e 4b8fc442-d8d26f5e 237827c9-d8d26f5e 5c9bda13-d8d26f5e 66d41c9d-d8d26f5e dcec42b9-d8d26f5e 45937cf-d8d26f5e dfb6ddc4-d8d26f5e 172199e7-d8d26f5e 24062188-d8d26f5e f6ee8563-d8d26f5e 9e562fe6-d8d26f5e 7089f363-d8d26f5e bd45ab20-d8d26f5e ec314542-d8d26f5e 1fcb7bba-d8d26f5e 54e86e30-d8d26f5e 3fabe990-d8d26f5e b8163a1b-d8d26f5e 58446682-d8d26f5e 39fc393d-d8d26f5e 646696ca-d8d26f5e a9e98ced-d8d26f5e 121ec64c-d8d26f5e 654bbccb-d8d26f5e 677e7ede-d8d26f5e cb80617e-d8d26f5e ca8b9f78-d8d26f5e a37d716f-d8d26f5e 381171fb-d8d26f5e b55f4b28-d8d26f5e fa505c3c-d8d26f5e 868a9bd9-d8d26f5e bcac1c59-d8d26f5e 7e17ac98-d8b87158 4a9a61df-d8b87158

</details>

**Comment by captainbrosset** [comment]
Thanks for the additional reports and error stack trace. This is still on our backlog and being actively investigated.

**Comment by captainbrosset** [comment]
Some updates from our end after researching this a bit more:

There are 2 issues at play here, manifesting in a similar way:

1. Lighthouse stays stuck with the message "Lighthouse is warming up..."
2. Lighthouse stays stuck with an exception displayed in the popup "Exception: could not create page handle for target tab".

Issue 1 happens in 116 (which is the current stable version of Edge). I announced [earlier](https://github.com/MicrosoftEdge/DevTools/issues/169#issuecomment-1686203704) that we had made a fix in 117. The plan was to cherry pick this fix to 116 too, but that unfortunately did not happen. The fix was correct for issue 1, but then issue 2 occured, which is why you all still had a broken Lighthouse.

Issue 2 happens in 117 and 118. We have a fix for this that's ready to roll out, and should be going to 118 (Canary and Dev) any time now. It will also make it to 117 (Beta) probably next week.

**You can enable this fix today yourself**: 

1. Open the `edge://flags` page.
2. In the search field, at the top, enter "DevTools using Tab Target".
3. Change the flag's value from "Default" to "Enabled".
4. Restart the browser.

Lighthouse should now work.
Note that this flag will be enabled by default in Edge very soon. So this is only if you want to use Lighthouse right now in Beta, Dev, or Canary.

So, bottom line is: it unfortunately looks like lighthouse is broken in stable and we won't be able to repair it there. But there's a fix you can do to get it to run in Beta, Dev, Canary. And that fix will be applied for you very soon (probably via the Edge updater in the September timeframe).

Note that Stable will soon become 117 (it's planned for 2023-09-14).

**Comment by dviate** [comment]
>Lighthouse should now work

Unfortunately it still shows me the warming up box here 😔

**Comment by dviate** [comment]
On random websites I want to test, I see this text in orange:
>There may be stored data affecting loading performance in this location: IndexedDB. Audit this page in an InPrivate window to prevent those resources from affecting your scores.

**Comment by XFox111** [comment]
> Lighthouse should now work.

I can confirm that Lighthouse is working on Canary build with `#devtools-tab-target` flag turned on. Doesn't work without it at the time though (throws `Could not create page handle for the target page` exception).

In case someone need it, here's the build info:
<details>
<summary><code>edge://version</code> on 118.0.2071.0 Canary</summary>

Microsoft Edge | 118.0.2071.0 (Official build) canary (64-bit)
-- | --
Revision | 44d110c744d144fec8837b4c075f92d766d2088f
Chromium version | 118.0.5967.0
Operating system | Windows 11 Version 22H2 (Build 22621.2215)
JavaScript | V8 11.8.6
User agent | Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/118.0.0.0 Safari/537.36 Edg/118.0.0.0
Command-line | "C:\Users\WDAGUtilityAccount\AppData\Local\Microsoft\Edge SxS\Application\msedge.exe" --restart --restart --flag-switches-begin --enable-features=DevToolsTabTarget --flag-switches-end
Executable path | C:\Users\WDAGUtilityAccount\AppData\Local\Microsoft\Edge SxS\Application\msedge.exe
Profile path | C:\Users\WDAGUtilityAccount\AppData\Local\Microsoft\Edge SxS\User Data\Default
Active Variations | 8fab779b-d8b87158 ea5e110f-d8b87158 711ef5c1-d8b87158 133488ad-d8b87158 14007ca6-d8b87158 4c14a889-d8b87158 cd0a2e45-ca7d8d80 ddef57c-d8b87158 8d005f24-d8b87158 d794b028-ca7d8d80 840681a4-d8b87158 1e92a065-d8b87158 5c33b2b4-d8b87158 b61cf9c4-d8b87158 a7896e11-ca7d8d80 a0596364-d8b87158 eb1bc628-d8b87158 ce2685d8-d8b87158 5b14374-d8b87158 7ae07155-ca7d8d80 18e0a776-ca7d8d80 3d684be3-d8b87158 ca0f6b5a-d8b87158 efe9c433-d8b87158 9cf91d25-ca7d8d80 17386f0-d8d26f5e b75782da-d8d26f5e c9615a18-d8b87158 360e8bdf-ca7d8d80 9ff0f0c2-d8b87158 4ea303a6-ca7d8d80 30028a54-d8b87158 5fe87c3b-d8b87158 afd40c01-d8b87158 5329947e-d8b87158 d01bec90-ca7d8d80 e9a1741-d8b87158 9210045f-d8b87158 1225f331-d8b87158 7a920913-d8b87158 2422dd57-d8b87158 18e1bdad-ca7d8d80 ce67070e-ca7d8d80 1969e5c-d8b87158 ffca31e9-d8b87158 793340c0-d8d26f5e 31e6ccb1-d8d26f5e f8f08c07-d8d26f5e f417542f-d8d26f5e 70ceb6a7-d8d26f5e b25b7c7e-d8d26f5e b1fa74fa-d8d26f5e ccdfedf0-d8d26f5e aaad07de-d8d26f5e cdf29b77-d8d26f5e 59aa47b7-d8d26f5e 80bdf841-d8d26f5e f174dd26-d8d26f5e af89d238-d8d26f5e 63a1a746-d8d26f5e e241c53-d8d26f5e d2280f33-d8d26f5e fbcd9197-d8d26f5e 2dab07dc-d8d26f5e 7accdcfa-d8d26f5e a408c5a6-d8d26f5e 4a8c62b4-d8d26f5e f5bba047-d8d26f5e 8891cb38-d8d26f5e a4ad23c5-d8d26f5e 611f1798-d8d26f5e 916bd790-d8d26f5e 8db5d24b-d8d26f5e 24e02e3-d8d26f5e 4f10b653-d8d26f5e a9140717-d8d26f5e 8287f006-d8d26f5e 86d34dfc-d8d26f5e b1508168-d8d26f5e 4c7ac06c-d8d26f5e b322f040-d8d26f5e c6fdbeb2-d8d26f5e 29bf163-d8d26f5e eaab553f-d8d26f5e cc4725b-d8d26f5e ed7b7a-d8d26f5e 1fc6c46e-d8d26f5e 69168e5a-d8d26f5e 4d4c330-d8d26f5e 4d3051ff-d8d26f5e f08cdbda-d8d26f5e db94232b-d8d26f5e 8f66cc60-d8d26f5e 80011f72-d8d26f5e e15deb37-d8d26f5e 72b74988-d8d26f5e daf8ad14-d8d26f5e dac2b8ab-d8d26f5e 851d4b5a-d8d26f5e c6a889f3-d8d26f5e e75f3d0d-d8d26f5e 4777da1c-d8d26f5e f510ede2-d8d26f5e 3f44e40-d8d26f5e d205504-d8d26f5e afc004b9-d8d26f5e 63e4737a-d8d26f5e 7edb7174-d8d26f5e f214853b-d8d26f5e d557113c-d8d26f5e e4305f1-d8d26f5e 4b8fc442-d8d26f5e 226439d5-d8d26f5e 237827c9-d8d26f5e 5c9bda13-d8d26f5e 66d41c9d-d8d26f5e 45937cf-d8d26f5e dfb6ddc4-d8d26f5e 172199e7-d8d26f5e 24062188-d8d26f5e 687bee50-d8d26f5e f6ee8563-d8d26f5e 9e562fe6-d8d26f5e 7089f363-d8d26f5e baad6c16-d8d26f5e bd45ab20-d8d26f5e ec314542-d8d26f5e 760043ac-d8d26f5e 1fcb7bba-d8d26f5e 54e86e30-d8d26f5e 3fabe990-d8d26f5e b8163a1b-d8d26f5e 58446682-d8d26f5e 39fc393d-d8d26f5e 646696ca-d8d26f5e ef45cf35-d8d26f5e a9e98ced-d8d26f5e 121ec64c-d8d26f5e 654bbccb-d8d26f5e 29f23bbb-d8d26f5e 67eef142-d8d26f5e 677e7ede-d8d26f5e 792a56ff-d8d26f5e aa74f0b0-d8d26f5e ca8b9f78-d8d26f5e a37d716f-d8d26f5e 381171fb-d8d26f5e b55f4b28-d8d26f5e fa505c3c-d8d26f5e 868a9bd9-d8d26f5e bcac1c59-d8d26f5e 4a9a61df-d8b87158

</details>

**Comment by Th3S4mur41** [comment]
Confirmed here too working in Edge Canary 118.0.2071.0 with [edge://flags/#devtools-tab-target](edge://flags/#devtools-tab-target) enabled

**Comment by robearlam** [comment]
Confirmed here as well.

Working in Edge 118.0.2060.1 with edge://flags/#devtools-tab-target enabled

**Comment by AngeloKarugo** [comment]
This has been resolved for me after updating to 116.0.1938.81 on stable build, without enabling any flags.
OS info:
Windows 10 Pro (build 19045.3393)

**Comment by mpotane** [comment]
Working for me as well thanks same build 116.0.1938.81

**Comment by captainbrosset** [comment]
Awesome, thank you for confirming. Closing this issue now.
